notebooks/week2/01_prepare_dataset_n.py

```diff
@@ -0,0 +1,41 @@
+# Databricks notebook source
+import logging
+import os
+
+from databricks.connect import DatabricksSession
+from pyspark.sql import SparkSession
+
+from taxinyc.config import ProjectConfig
+from taxinyc.data_processor import DataProcessor
+
+spark = SparkSession.builder.getOrCreate()
+logging.basicConfig(level=logging.INFO, format="%(asctime)s - %(levelname)s - %(message)s")
+logger = logging.getLogger(__name__)
+
+# COMMAND ----------
+
+# path stuff is annoying, in vs code it seems like everything gets executed from repo main level
+try:
+    config = ProjectConfig.from_yaml(config_path="project_config.yml")
+except Exception:
+    config = ProjectConfig.from_yaml(config_path="../../project_config.yml")
+
+spark = DatabricksSession.builder.profile(os.environ["DATABRICKS_PROFILE"]).getOrCreate()
```

Comment:
better to use SparkSession, and no profile definition - this code should run anywhere